omnibus_overrides.rb

```diff
@@ -10,10 +10,10 @@
 override "libyaml", version: "0.1.6"
 override "makedepend", version: "1.0.5"
 override "ncurses", version: "5.9"
-override "openssl", version: "1.0.2h"
 override "pkg-config-lite", version: "0.28-1"
 override "ruby", version: "2.3.1"
 override "ruby-windows-devkit-bash", version: "3.1.23-4-msys-1.0.18"
 override "util-macros", version: "1.19.0"
 override "xproto", version: "7.0.28"
 override "zlib", version: "1.2.8"
+override "openssl", version: "1.0.2h"
```

Comment:
Out of curiosity, you changed the order of the override for any particular reason? 🤔